ui/js/values.js

```diff
@@ -53,7 +53,9 @@ treeherder.value("thPlatformMap", {
     "android-4-0-armv7-api10": "Android 4.0 API10+",
     "android-4-0-armv7-api11": "Android 4.0 API11+",
     "android-4-0-armv7-api15": "Android 4.0 API15+",
+    "android-4-0-armv7-api15-old-id": "Android 4.0 API15+ OldId",
     "android-4-2-x86": "Android 4.2 x86",
+    "android-4-2-x86": "Android 4.2 x86 OldId",
```

Comment:
The key name is a duplicate of the one used in the line above it - I'm presuming it was meant to have the `-old-id` suffix?